Help/MAC_Hints.rst

```diff
@@ -13,7 +13,15 @@ Coverage Installation Hints for macOS Users:
     $ brew install python3
     $ python3 --version   # To check the installed version
 
-3. Create Virtual Environments with pyvenv
+3. Install pyenv with homebrew
```

Comment:
This should be instead of the section "2. Install Python3."

See https://gist.github.com/Bouke/11261620#gistcomment-1573091